### PR TITLE
add link and restriction to nav

### PR DESCRIPTION
**Pull request by LeeRenJie**
depends on #126

### CODE REVIEW

File: finance-tracker/app/views/portfolio/index.html.erb [hunk 1]
```diff
@@ -17,3 +17,8 @@
 </div>
 <div id="results"></div>
 
+<% unless @tracked_stocks.empty? %>
+  <%= render 'stocks/list' %>
+<% end %>
+
+
```
Comment (tlkiong) [inline finance-tracker/app/views/portfolio/index.html.erb:24]: ?

File: finance-tracker/app/views/stocks/_list.html.erb [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+          <th scope="col">Name</th>
+          <th scope="col">Price</th>
+          <th scope="col">Action</th>
+        </tr>
+      </thead>
+      <tbody>
+        <% @tracked_stocks.each do |stock| %>
+          <tr>
+            <th scope="row"><%= stock.ticker %></th>
+            <td><%= stock.name %></td>
```
Comment (tlkiong) [inline finance-tracker/app/views/stocks/_list.html.erb:16]: Why use `th` in `tr`?

**Review by tlkiong — CHANGES_REQUESTED** [review]

File: finance-tracker/app/views/portfolio/index.html.erb (hunk 1 above)
Comment (LeeRenJie) [reply finance-tracker/app/views/portfolio/index.html.erb]: `tracked_stocks_empty?` returns true if no stocks are tracked. In this case, we don't want to render the list partial. So use `unless` to replace `if !tracked_stocks_empty?` to only render if there are stocks to list.

File: finance-tracker/app/views/stocks/_list.html.erb (hunk 2 above)
Comment (LeeRenJie) [reply finance-tracker/app/views/stocks/_list.html.erb]: I think the tutorial wants it to be in bold 😅 I'll use the bold tag then.

File: finance-tracker/app/views/portfolio/index.html.erb (hunk 1 above)
Comment (tlkiong) [reply finance-tracker/app/views/portfolio/index.html.erb]: I jz tested you whether you pay attention to the details not.
my `?` is on the **EXTRA** empty line you have in this file.

File: finance-tracker/app/views/stocks/_list.html.erb [hunk 3]
```diff
@@ -12,7 +12,7 @@
       <tbody>
         <% @tracked_stocks.each do |stock| %>
           <tr>
-            <th scope="row"><%= stock.ticker %></th>
+            <td scope="row"><b><%= stock.ticker %></b></td>
```
Comment (tlkiong) [inline finance-tracker/app/views/stocks/_list.html.erb:15]: What is `scope="row"` means?

**Review by tlkiong — CHANGES_REQUESTED** [review]
.

File: finance-tracker/app/views/stocks/_list.html.erb (hunk 3 above)
Comment (LeeRenJie) [reply finance-tracker/app/views/stocks/_list.html.erb:15]: specifies if the header is for a row or column. I will remove this.

**Review by tlkiong — APPROVED** [review]

**Pull request merged**